elytron/src/test/java/org/wildfly/extension/elytron/CredentialStoreUtility.java

```diff
@@ -95,7 +95,7 @@ public CredentialStoreUtility(String credentialStoreFileName, String storePasswo
             LOGGER.error(t);
             throw new RuntimeException(t);
         }
-        LOGGER.debugf("Credential Store created [%s] with master password \"%s\"", credentialStoreFileName, storePassword);
+        LOGGER.debugf("Credential Store created [%s] with primary password \"%s\"", credentialStoreFileName, storePassword);
```

Comment:
So far the concept of primary/secondary is just for "Domain Model", see WFLY-14053. +1 for discovering this, but this one would need to be treated differently. 

@fjuma Do you have any suggestions for this word?

Reply:
perfect, thanks @fjuma

Reply:
master is dropped from logger